acceptance/srv-configs/clusterman.yaml

```diff
@@ -44,7 +44,8 @@ autoscaling:
     target_capacity_margin: 0.05
 
 autoscale_signal:
-    internal: True
+    name: ConstantSignal
+    branch_or_tag: acceptance
```

Comment:
I think you changed this since the last review? Does this make it so the tests don't use the internal pending pods sensitive signal?

Reply:
This means that the default signal is not using the internal pending pods signal, but the pool-configured signal still is.  The default signal is just used in case the pool signal is not present or doesn't exist.  It's necessary to change this because the Mesos-specific itests are using the same base configuration files as the Kubernetes-specific itests (but they're using different pool config files).